src/graphorm/graphorm.ts

```diff
@@ -677,6 +677,7 @@ export class GraphORM {
    * @param nodeToCursor A function that creates a cursor from a node
    * @param beforeQuery A callback function that is called before executing the query
    * @param transformNodes Apply any transformation on the nodes before adding page info
+   * @param readReplica Whether to use the read replica instance
```

Comment:
This one is basically the same as: https://github.com/dailydotdev/daily-api/pull/2286